src/dns_sd.cpp

```diff
@@ -86,25 +86,27 @@ init(Handle<Object> target) {
     addConstants(target);
 }
 
-inline
+NAN_INLINE
```

Comment:
It simply makes V8_INLINE consistently available. Using it is optional. Besides, it is too late to remove it now, a bunch of code would break for no good reason.